tox.ini

```
@@ -0,0 +1,94 @@
+# Global tox file
+
+# This file is used to invoke tox in individual charms
+
+[tox]
+skipsdist = True
+envlist = pep8,py3
```

This isn't the correct usage of `envlist` here. `envlist` is used to declare the default environments that will be run if the user does not pass any environment as a flag to `tox run`: https://tox.wiki/en/latest/config.html#env_list

`fmt`, `lint`, `type`, and `unit` can be in the default envlist since they are not beefy like the integration tests.

```suggestion
envlist = fmt,lint,type,unit
```